index.html

```diff
@@ -0,0 +1,154 @@
+            </article>
+
+            <article class="grid-article">
+                <img src="./assets/icon_web-development.svg" alt="Web Development img">
+                <h3>Web Development</h3>
+                 <p>Modern websites need to be responsive and fully functional at all sizes. We develop all of our sites with a mobile first approach and ensure that our code quality is clean, organized and performant. Already have a design but need it developed? We’re here to help!</p>
+            </article>
+
+            <article class="grid-article">
+                <img src="./assets/icon_ux.svg"  alt="User Experience img">
+                <h3>User Experience</h3>
+                <p> In order to build successful products you have to understand the user. We create user personas, wireframes and prototypes to test with users. This allows us to gather feedback which helps to shape the product and can quickly eliminate potential risks – saving you time and money.</p>
+            </article>
+
+            <article class="grid-article">
+                <img src="./assets/icon_graphic-design.svg" alt="Logo Design + Branding img">
+                <h3>Logo Design + Branding</h3>
+                <p> Nike, Apple, Coca-Cola, Starbucks. What do all of these brands have in common? Easily recognizable logos and brand identity. The best brands stand out from the crowd by having a clear and unique point of view. We will work with you to understand your vision and help create a unique brand identity and style guide for your company.</p>
+            </article>
+
+            <article class="grid-article">
+                    <img src="./assets/icon_seo.svg" alt="SEO img">
+                    <h3>SEO</h3>
+                    <p>Do you want your site to rank higher in search results? Google is constantly changing their algorithms for how they rank pages. Don’t worry about constantly playing catch-up. We will make sure your website it optimized, providing you the best opportunities for organic search exposure.</p>
+            </article>
+
+
+            <article class="grid-article">
+                    <img src="./assets/icon_copywriting.svg" alt="Copy writing img">
+                    <h3>Copy Writing</h3>
+                    <p>Brand messaging needs to be clear, direct and easy to understand to be effective. A unique tone of voice will help you stand out from the crowd and build brand loyalty. We work with you to establish your brand bible and ensure your messaging is focused and targeted.</p>
+            </article>
+
+        </div>
+    </section>
+
+    <section class="outer-container form-section">
+        <div class="inner-container form">
+            <form action="">
```

Comment:
Het form wil je een class geven zodat je alle children van het form element netjes in het midden kan placeren. Hiermee wordt dus ook de button gecentreerd!